public/components/getting_started/components/getting_started_collectData.tsx

```diff
@@ -349,20 +349,20 @@ export const CollectAndShipData: React.FC<CollectAndShipDataProps> = ({
           <EuiFlexItem>
             <EuiCheckableCard
               id="use_sample_dataset"
-              label="Use a sample dataset"
+              label="Configure use-case based content"
               checkableType="radio"
-              checked={selectedCard === 'Use a sample dataset'}
+              checked={selectedCard === 'Configure use-case based content'}
```

Comment:
not blocking issue, might be better to use id instead of user-facing text to check, otherwise every time this text changes this check also needs to change

Reply:
Added in id as cardOne, cardTwo, cardThree.
Thanks.

Reply:
instead of cardOne, cardTwo, cardThree, maybe use an array? and checking user-facing text won't work if we introduce i18n translations. usually this is done by checking `id` or `key` instead of user-facing text